src/zc/table/tests.py

```diff
@@ -65,20 +67,44 @@ def fieldColumnSetUp(test):
         zope.formlib.interfaces.IDisplayWidget)
 
 
+# Strip out u'' literals in doctests, adapted from
+# <https://stackoverflow.com/a/56507895>.
+class Py23OutputChecker(doctest.OutputChecker, object):
+
+    RE = re.compile(r"(\W|^)[uU]([rR]?[\'\"])", re.UNICODE)
+
+    def remove_u(self, want, got):
+        return (re.sub(self.RE, r'\1\2', want),
+                re.sub(self.RE, r'\1\2', got))
+
+    def check_output(self, want, got, optionflags):
+        want, got = self.remove_u(want, got)
+        return super(Py23OutputChecker, self).check_output(
+            want, got, optionflags)
+
+    def output_difference(self, example, got, optionflags):
+        example.want, got = self.remove_u(example.want, got)
+        return super(Py23OutputChecker, self).output_difference(
+            example, got, optionflags)
+
+
 def test_suite():
-    import doctest
     return unittest.TestSuite((
-        doctest.DocFileSuite('README.txt',
+        doctest.DocFileSuite(
+            'README.txt',
             optionflags=doctest.NORMALIZE_WHITESPACE+doctest.ELLIPSIS,
```

Comment:
(It hurts me to see bit flags combined with a `+` rather than a `|`.  Unrelated to this PR but adjacent to other changes in it, so maybe you could ... )

Reply:
done